tests/integration/components/paper/date-picker-test.js

```diff
@@ -0,0 +1,223 @@
+import hbs                        from 'htmlbars-inline-precompile';
+import { module, test }           from 'qunit';
+import { render, find, focus,
+  fillIn, blur, triggerKeyEvent } from '@ember/test-helpers';
+import { setupRenderingTest }     from 'ember-qunit';
+
+module('Integration | Component | {{paper/date-picker}}', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.set('value', new Date(2018, 9, 10, 0, 0));
+    this.actions = { update: (value) => {
+      this.set('value', value);
+    } };
+  });
+
+  test('date gets formatted to date string', async function(assert) {
+    this.set('value', new Date(2015, 0, 1));
+    await render(hbs`{{paper/date-picker value=value updated=(action "update")}}`);
+    let $input = find('input');
+    assert.equal($input.value, '01-01-15');
```

Comment:
Have the day different from the moth, so you know which order it will be formatted. Now it could say month-day-year and still pass.